README.md

```diff
@@ -0,0 +1,18 @@
+# Smoothers
+
+Visualizations of various one-dimensional smoothers using the d3 javascript library. This is an excellent tool to play with different models for one dimensional regresssion setting and understand the bais-variance trade off key concept.
```

Comment:
:+1: Just a few knits to pick
`different models for one dimensional` --> `different models in the one-dimensional` 
`regresssion` --> `regression`
`bais` --> `bias`